### ISSUE TITLE
Transformer bug (duplicating energy?)

### ISSUE THREAD

**Issue by TheYxel** [TinyModularThings/IC2Classic#763]
When you connect TR (transformer), connected to ES (energy storage) back to ES it begins transmit packets to it. 
When you connect some energy user to that TR, TR splits packets and send half of them back to ES.
So Mass Fabricator (I tried other energy users, no differ) gets 1024 EU/t instead of 2048 Eu/t. That other 1024 Eu goes back to ES.

![mc1](https://github.com/TinyModularThings/IC2Classic/assets/131541322/8e454fef-9877-4592-adf9-2a73707c4c50)
![mc2](https://github.com/TinyModularThings/IC2Classic/assets/131541322/22c915af-3131-4aba-a5a9-c62244a74abf)
![mc3](https://github.com/TinyModularThings/IC2Classic/assets/131541322/6425cdf9-8018-40ab-a317-9b2daf13923d)

**Comment by Meduris** [comment]
I don't see how this would be duplicating energy, you literally split up 2048 eu/packet packages up to 4 512 eu/packet packets with the transformer and those get distributed across different machines, the total the other machines receive is still just 2048 eu in total.
Your screenshots also show exactly that, with the ESU and Mass Fab both receiving 1024 eu (so both get 2 512eu packets) while the ESU outputs 2048 eu.

So basically as far as I can tell from your screenshots, the only thing happening is that you loop back half of the energy packets back to your ESU due to the extra cable connection instead of fully powering your Mass Fab with it.

**Comment by Speiger** [comment]
@TheYxel as @Meduris said not a bug all intended behavior.
